docs/fr/guides/actions.md

```diff
@@ -193,6 +193,19 @@ entry: send({ type: 'SOME_EVENT' });
 
 ## Send action
 
+::: warning
```

Comment:
Those warnings should probably mention `raise` as alternative for sending to "self".